core-java-modules/core-java-nio/README.md

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
For brand new modules, we don't have to create this file up-front. The editor will add this file once the article is published, as the final URL won't be determined until then.

Reply:
ok. i have removed the readme.md file.